### PR TITLE
Add blurb about `svix-react` library

### PR DESCRIPTION
**Pull request by svix-liam**
image

### CODE REVIEW

File: docs/management-ui.mdx [hunk 1]
```diff
@@ -93,6 +93,18 @@ To add this to your application, just pass the URL you received in the previous
...
+We also provide the [`svix-react`](https://www.npmjs.com/package/svix-react) package that you can use to easily embed the App Portal. To use it, simply run:
+
+```sh
+npm install svix-react
+# or
+yarn add svix-react
+```
+
+The npm package is lightweight and provides some basic styling and a loading indicator out of the box. For a more complete usage example, check out the [package README](https://www.npmjs.com/package/svix-react).
+
```
Comment (tasn) [inline docs/management-ui.mdx:107]: Missing a very simple React code sample.

Comment (svix-liam) [reply docs/management-ui.mdx:140]: Added

File: docs/management-ui.mdx [hunk 2]
```diff
@@ -93,6 +93,51 @@ To add this to your application, just pass the URL you received in the previous
...
+```sh
+npm install svix-react
+# or
+yarn add svix-react
+```
+
+The npm package is lightweight and provides some basic styling and a loading indicator out of the box. To use it, simply provide the magic link from the [`get_dashboard_access` endpoint](https://api.svix.com/docs#operation/get_dashboard_access_api_v1_auth_dashboard_access__app_id___post).
+
+#### Example
```
Comment (tasn) [inline docs/management-ui.mdx:108]:
```suggestion
```

**Review by tasn — APPROVED** [review]
No need for this subtitle. Other than that it can go in.

**Pull request merged**